workspaces/arborist/test/arborist/reify.js

```diff
@@ -4,36 +4,22 @@ const runScript = require('@npmcli/run-script')
 const localeCompare = require('@isaacs/string-locale-compare')('en')
 const tnock = require('../fixtures/tnock')
 
-// mock rimraf so we can make it fail in rollback tests
-const realRimraf = require('rimraf')
-let failRimraf = false
-const rimrafMock = (...args) => {
-  if (!failRimraf) {
-    return realRimraf(...args)
-  } else {
-    return args.pop()(new Error('rimraf fail'))
-  }
-}
-rimrafMock.sync = (...args) => {
-  if (!failRimraf) {
-    return realRimraf.sync(...args)
-  } else {
-    throw new Error('rimraf fail')
-  }
-}
 const fs = require('fs')
+
+let failRm = false
 let failRename = null
 let failRenameOnce = null
 let failMkdir = null
-const { rename: realRename, mkdir: realMkdir } = fs
+const { rename: realRename, rm: realRm, mkdir: realMkdir } = fs
```

Comment:
are we doing other things that claim the `rm`, `rename`, and `mkdir` namespaces?

Reply:
i don't think so. i did it this way in the interest of minimal changes